src/Uno.UI/UI/Xaml/UIElement.Layout.netstd.cs

```diff
@@ -118,13 +118,13 @@ private void InvalidateArrangeDirtyPath()
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		private void InvalidateParentArrangeDirtyPath()
 		{
-			if (this.GetParent() is UIElement parent)
+			if (this.GetParent() is UIElement parent) //TODO:MZ: Should this use VisualTree.GetParent as fallback?
```

Comment:
Same here

Reply:
Closing in favor of the conversation above, but included this in the Epic